### DISCUSSION TITLE
Docs on Collections and File paths

### DISCUSSION THREAD

**Discussion by Jeppedy** [manyfold3d/manyfold#3172] [Support] [answered]
Is there documentation somewhere that describes how collects work with file paths?
I often have a couple of levels housing my files for folder-based organization:
/(Project name, like Gridfinity)/(grouping, like Bins or Bases)/(type, like standard, custom, etc)

What's the best way to approach this in MF?

**Comment by Floppy** [comment, marked as answer]
There isn't a good document on it yet, I really need to write something I think. There are two parts to it:

1) Manyfold will organise files it knows about into folders etc based on the template in the site settings - so you can say you want `/{collection}/{modelName}{modelId}` for instance, which will organise on disk into collection folders. I *think* that handles nested collections, but I'd have to check.

2) Optionally, when scanning a folder on disk for existing files, you can get it to try to work out what the collections etc *should* be from that same template. By default that template will treat everything except the model name as a tag, I think.

And by explaining that, I've just realised a problem with number 2 - when you first set up, you can't change the template before it starts scanning, so everything will always come in as tags initially. That's something I should improve! I'd guess you want to set that template when you set up the first library, so that it can parse the files into a sensible structure first time. See, this is why I really should write the documentation, I realise things that are obvious in hindsight!

**Reply by Floppy** [reply]
I've added that as a feature in #3192

**Comment by Jeppedy** [comment]
This is an absolutely amazing application. You are doing fantastic work
with it and we appreciate it. I expected the app to be thrown together and
only a half solution. But you have done an amazing job of solving this
problem for us all. If there's anything I can do to help, please reach out.

**Reply by Floppy** [reply]
Aw, thank you ❤️
